packages/rest/src/lib/RequestManager.ts

```diff
@@ -54,13 +54,19 @@ export interface RequestData {
 	 */
 	authPrefix?: 'Bot' | 'Bearer';
 	/**
-	 * The body to send to this request
+	 * The body to send to this request.
+	 * If providing as BodyInit, set `passThroughBody: true`
 	 */
-	body?: unknown;
+	body?: BodyInit | unknown;
 	/**
 	 * Additional headers to add to this request
 	 */
 	headers?: Record<string, string>;
+	/**
+	 * Whether to pass-through the body property directly to `fetch()`.
+	 * <warn>This only applies when attachments is NOT present</warn>
+	 */
+	passThroughBody?: boolean;
```

Comment:
Passthrough is one word.

Reply:
Isn't it both? Where passthrough is used as a noun, e.g. a passthrough panel, and pass through / pass-through is used like this to say the body is allowed to pass through.

I did struggle with capitalization on this, so want to make sure its right though.

Reply:
It is both, as a fact, Node.js has a class named [`PassThrough`](https://nodejs.org/api/stream.html#class-streampassthrough).